### PR TITLE
Group waterings by garden

### PR DESCRIPTION
**Pull request by bendelonlee**
DISCLAIMER: DO NOT let this PR add stress. No need to read if you'd rather focus on the project. 

Each garden is now in its own div inside each day.  I struggled for a while with how to do this, it turns out that `includes` more or less takes the scope of your AR query (Did we all know this? Maybe it was mentioned, but I didn't fully grok it until now), which was very useful. 

We can add colors if we want! We'll also probably want to style the divs. Spacing and all that. Right now they're just divs. 

If people don't have time to review, we can keep pushing up the same branch and them merge with master later. Or whatever really. It's all good.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -42,6 +42,7 @@ gem 'twilio-ruby', '~> 5.20.1'
 gem 'sidekiq'
 gem 'newrelic_rpm'
 gem 'capybara-email'
+gem 'table_print'
```
Comment (asmolentzov) [inline Gemfile]: what is this magical gem?

File: app/assets/javascripts/schedule.js [hunk 2]
```diff
@@ -20,7 +20,6 @@ $( document ).ready(function() {
     $( ".droppable" ).droppable({
       tolerance: 'touch',
       drop: function( event, ui ) {
-        debugger;
```
Comment (asmolentzov) [inline app/assets/javascripts/schedule.js]: haha, good remembering to remove this!

**Comment by bendelonlee** [comment]
Added commits for drag and drop.

File: Gemfile (hunk 1 above)
Comment (asmolentzov) [reply Gemfile]: Per Ben - this helps print ActiveRecord queries in a table format

Comment (le3ah) [reply Gemfile]: 🎉

File: app/assets/javascripts/schedule.js (hunk 2 above)
Comment (le3ah) [reply app/assets/javascripts/schedule.js]: 🐞

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -13,6 +13,13 @@ body {
   opacity: 0.9;
 }
 
+a {
+  color: royalblue;
+  &:hover {
```
Comment (le3ah) [inline app/assets/stylesheets/application.scss]: fancy!

File: app/facades/garden_facade.rb [hunk 4]
```diff
@@ -4,9 +4,9 @@ def initialize(user)
   end
 
   def weather(garden)
-    @_weather ||= Weather.new(garden)
+    Weather.new(garden)
   end
```
Comment (le3ah) [inline app/facades/garden_facade.rb]: Bye, memoization! 👋

File: app/assets/javascripts/schedule.js [hunk 5]
```diff
@@ -1,33 +1,47 @@
```
Comment (abenetka) [inline app/assets/javascripts/schedule.js]: Magic!

File: app/assets/javascripts/schedule.js [hunk 6]
```diff
@@ -1,33 +1,47 @@
...
-        let id = ui.draggable[0].id
-        let field = `#${id}-water-time`
-        let date = this.parentElement.attributes.name.nodeValue
-        $(field).val(date)
-        $(`#update-${id}`).click();
+  $( ".droppable" ).droppable({
+    tolerance: 'touch',
+    drop: function( event, ui ) {
+      let draggedFrom = $(ui.draggable[0]).parent();
+      let garden_classes = draggedFrom.attr('class');
```
Comment (abenetka) [inline app/assets/javascripts/schedule.js]: magic!

File: app/views/users/_phone_number_form.html.erb [hunk 7]
```diff
@@ -0,0 +1,9 @@
+<div class="phone">
```
Comment (abenetka) [inline app/views/users/_phone_number_form.html.erb]: Nice!

**Pull request merged**